File: app/src/main/java/com/drdisagree/colorblendr/utils/annotations/TestingOnly.java
package com.drdisagree.colorblendr.utils.annotations;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 * Annotation to mark fields and methods used for testing purposes only.
 */
@Retention(RetentionPolicy.SOURCE)
@Target({ElementType.FIELD, ElementType.METHOD})
public @interface TestingOnly {
}

File: app/src/main/java/com/drdisagree/colorblendr/utils/annotations/TestingOnly.kt
package com.drdisagree.colorblendr.utils.annotations

/*
 * Annotation to mark fields and methods used for testing purposes only.
 */
@Retention(AnnotationRetention.SOURCE)
@Target(
    AnnotationTarget.FIELD,
    AnnotationTarget.FUNCTION,
    AnnotationTarget.PROPERTY_GETTER,
    AnnotationTarget.PROPERTY_SETTER
)
annotation class TestingOnly 
